content/guards.md

```diff
@@ -125,6 +127,7 @@ Global guards are used across the whole application, for every controller and ev
 @@filename(app.module)
 import { Module } from '@nestjs/common';
 import { APP_GUARD } from '@nestjs/core';
+import { RolesGuard } from './roles.guard';
```

Comment:
Having `roles.guard.ts` file at the same level as `app.module` does not seem to be an ideal solution, that's why we skipped import (to let developers decide where to put this class).

Reply:
I understand the reasoning behind giving freedom of where to store the module, but the actual example may introduce a confusion regarding whether the module is injected in some "magical" way since there is no import mentioned. 
Maybe we could go with? 
```suggestion
import { RolesGuard } from './path-to-your-roles.guard';
```